src/core.ts

```diff
@@ -76,10 +76,27 @@ function getExposedContent(ast: SourceUnit, inputPath: string, contractMap: Cont
           contractHeader.push(`is ${c.name}`);
         }
         contractHeader.push('{');
+
         return [
           contractHeader.join(' '),
           spaceBetween(
             makeConstructor(c, contractMap),
+            ...getInternalVariables(c, contractMap).map(v => {
+              const header = [
+                'function',
+                `x${v.name}(${getGetterTypes(v).map(a => `${a.type} ${a.name}`).join(', ')})`,
+                'external',
+                'view',
+                'returns',
+              ];
+              header.push(`(${getGetterValues(v).map(a => `${a.type}`)})`);
+              header.push('{');
+              return [
+                header.join(' '), [
+                  `return ${v.name}` + (getGetterTypes(v).length > 0 ? `${getGetterTypes(v).map(a => `[${a.name}]`).join('')}` : '') + ';',
```

Comment:
You don't really need the conditional here. If length is 0, the `map(...).join('')` will by itself result in an empty string.